packages/blockchain/examples/4444.ts

```diff
@@ -0,0 +1,87 @@
+import { createBlock } from '@ethereumjs/block'
+import { createBlockchain } from '@ethereumjs/blockchain'
+import { Common, Hardfork, Mainnet } from '@ethereumjs/common'
+import { bytesToHex, hexToBytes } from '@ethereumjs/util'
+
+const main = async () => {
+  const common = new Common({ chain: Mainnet, hardfork: Hardfork.Shanghai })
+  // Use the safe static constructor which awaits the init method
+  const blockchain = await createBlockchain({
+    validateBlocks: false, // Skipping validation so we can make a simple chain without having to provide complete blocks
```

Comment:
Ok, so the `validateBlocks` flag now also allows to insert blocks not being in the canoncial order, is this correct? (or was this behavior already there bofore) I generally do like this! 🙂 

So, and - otherway around - if `validateBlocks: true`, then it remains forbidden to not go the normal path, right (or not), so 1,2,3,4,...?

Reply:
Or, not sure if I am fully like it: can't we just fully validate again after this one not validated block (15537393n + 500n). So isn't it only this one block which can't be validated (and therefore would be something like a checkpoint) and then we can (and should) turn validation on again? Or can we do this with the current API already in this example?

Reply:
i just picked the flag from other example, but i think yes we can enable the flag, and yes if the parent is missing then it won't be validated, so optimistic inserts will happen

but when you have put an anchor and move the head forward (which again happens through the forward putblocks along the backfilled chain), then they should be validated

i will remove this from the example and see if it works (most probably should)

Reply:
Ah, forget about this, this is just simplification for the example, right?